components/raftstore/src/store/worker/pd.rs

```diff
@@ -733,16 +728,12 @@ where
 
         let mut report_peers = HashMap::default();
         for (region_id, region_peer) in &mut self.region_peers {
-            let read_bytes = region_peer.read_bytes - region_peer.last_store_report_read_bytes;
-            let read_keys = region_peer.read_keys - region_peer.last_store_report_read_keys;
-            let read_query_stats = region_peer
-                .read_query_stats
-                .sub_query_stats(&region_peer.last_store_report_query_stats);
-            region_peer.last_store_report_read_bytes = region_peer.read_bytes;
-            region_peer.last_store_report_read_keys = region_peer.read_keys;
-            region_peer
-                .last_store_report_query_stats
-                .fill_query_stats(&region_peer.read_query_stats);
+            let read_bytes = region_peer.store_report_read_bytes;
+            let read_keys = region_peer.store_report_read_keys;
+            region_peer.store_report_read_bytes = 0;
+            region_peer.store_report_read_keys = 0;
+            let read_query_stats = region_peer.store_report_read_stats.clone();
```

Comment:
how about `mem::swap(&mut region_peer.store_report_read_stats.0, &mut read_query_stats)`?

Reply:
not necessary I think.

Reply:
it will reduce clone